### ISSUE TITLE
Need a factor to reduce PAR from visible solar for I compset cases

### ISSUE THREAD

**Issue by ekluzek** [ESCOMP/CTSM#1713]
This is something that Rong Li brought to our attention in her work on the SIF code. There are two issues. One is that in datm we divide solar radiation evenly between the visible and near-IR bands using a ratio of 0.5. The other is that PAR is only part of the visible spectrum and not the entire amount.

The ratio between NIR and VIS is done for the offline model in datm. The ratio based on FluxNet sites is more like 0.43. The justification for this ratio in datm given in a comment is:

``` fortran
          ! relationship between incoming NIR or VIS radiation and ratio of
          ! direct to diffuse radiation calculated based on one year's worth of
          ! hourly CAM output from CAM version cam3_5_55
```

However, if we modify NIR and VIS in the datm, that will have a whole list of downstream affects on the model that would likely be a big tuning exercise to workout.

@RongLi29 way to resolve this was to apply a factor inside of photosynthesis that only applies for what's being used as PAR. The factor is only appropriate when running I cases (with datm rather than CAM). So we'll need to have it apply for I cases, but not B or F cases.

@RongLi29 @danicalombardozzi

**Comment by ekluzek** [comment]
This is the modification that @RongLi29 made to SurfaceRadiationMod.F90 to adjust for this for the offline I cases:

``` fortran
           ! for leaf APAR
           parlsun_z(p,iv) = forc_solad(g,ipar)*fabdl_sun_z(p,iv)*0.87 + forc_solai(g,ipar)*fabil_sun_z(p,iv)*0.87
           parlsha_z(p,iv) = forc_solad(g,ipar)*fabdl_sha_z(p,iv)*0.87 + forc_solai(g,ipar)*fabil_sha_z(p,iv)*0.87

```

**Comment by dlawrenncar** [comment]
Regarding the comment about using the 'correct' NIR / VIS ratio being a big
tuning exercise, is that true?   And do we want to base our decision not to
use the 'better' number on the possibility that it will result in a tuning
exercise?  It seems like we should at least consider whether or not we
would be better off in the long run if we modified the ratio according to
observations.  We could at least run a test simulation to see how big the
impact would be.  Also, and related, what is the ratio coming out of CAM.
If the 0.43 value is closer to what CAM produces, then that reduces the
impact of switching between CAM and DATM.

Note that the comment that @Erik Kluzek ***@***.***> pointed to refers
to the partitioning of direct and diffuse radiation whether that be NIR or
VIS (I think).  That 'parameterization' should be updated to account for
solar zenith angle, but that is a separate issue, I think.

**Comment by ekluzek** [comment]
Here's the entire bit of code from CDEPS that gives this when only SW down is provided. You are right the ratio between direct and diffuse is 0.5, the ratio between near-IR and visible is more complex.

 ``` fortran
         swndr = strm_swdn(n) * 0.50_r8
          ratio_rvrf =  min(0.99_r8,max(0.29548_r8 + 0.00504_r8*swndr  &
               -1.4957e-05_r8*swndr**2 + 1.4881e-08_r8*swndr**3,0.01_r8))
          Faxa_swndr(n) = ratio_rvrf*swndr
          swndf = strm_swdn(n) * 0.50_r8
          Faxa_swndf(n) = (1._r8 - ratio_rvrf)*swndf

          swvdr = strm_swdn(n) * 0.50_r8
          ratio_rvrf =   min(0.99_r8,max(0.17639_r8 + 0.00380_r8*swvdr  &
               -9.0039e-06_r8*swvdr**2 + 8.1351e-09_r8*swvdr**3,0.01_r8))
          Faxa_swvdr(n) = ratio_rvrf*swvdr
          swvdf = strm_swdn(n) * 0.50_r8
          Faxa_swvdf(n) = (1._r8 - ratio_rvrf)*swvdf
```

If I've got this right it looks like it starts out more evenly divided for low SW levels, but as SW increases it goes to the threshold of 99% visible and 1% near-IR for full sun.

**Comment by dlawrenncar** [comment]
The ratio_rvrt term is adjusting only the direct vs diffuse fraction.  My
understanding is that Rong is suggesting that the 0.5 value is modified to
0.43, controlling the total VIS (direct + diffuse) vs the total NIR
(direct + diffuse).  Again, that calculation for the ratio_rvrf could and
probably should be modified, but that is a separate, though perhaps
somewhat related, topic.

**Comment by RongLi29** [comment]
Yeah,  my understanding is shortwave is evenly split into VIS and NIR, and the ratio between direct and diffuse is the more complex one (the polynomial). From the code: NIR direct + NIR diffuse = Faxa_swndr+Faxa_swndf = strm_swdn * 0.5; similarly, total VIS = Faxa_swvdr+Faxa_swvdf = strm_swdn * 0.5.

My rough estimate based on FLUXNET data showed the ratio between PAR and SR is closer to 0.43 (see figure below). I would also note that in [Ryu et al (2018](http://dx.doi.org/10.1016/j.rse.2017.09.021)), BESS and GLASS gave global average PAR/SR of 0.46 (they also reported slight positive bias of SR and negative bias of PAR, which may partially explain the difference between their ratio and what I derived from fluxnet), and[ Tsubo and Walker (2005)](https://doi.org/10.1007/s00704-004-0080-5) has a summary of PAR/SR reported at different locations ranging from 0.44-0.48 if PAR is defined as 0.4-0.7 μm. So there might be some discussion on which is the proper value to use, but 0.5 seems to be on the high end.

![PAR_SR_hh](https://user-images.githubusercontent.com/54550132/163893228-8a1e55e4-4eca-4285-a18c-ae9a3499e2c4.png)

The code here seems consistent with the CLM5 tech note by the way:

![image](https://user-images.githubusercontent.com/54550132/163889874-7ffc0482-1c31-48b8-b45b-d8e569b434d8.png)

**Comment by wwieder** [comment]
I don't think this is a high priority at this point
